wire-ios/Wire-iOS/Sources/UserInterface/Settings/CellDescriptors/SettingsCellDescriptorFactory+Developer.swift

```diff
@@ -98,7 +98,7 @@ extension SettingsCellDescriptorFactory {
                            presentationAction: { DatabaseStatisticsController() })
         )
 
-        if !Analytics.shared.isOptedOut && !TrackingManager.shared.disableAnalyticsSharing {
+        if !trackingManager.disableAnalyticsSharing {
```

Comment:
I wonder what role the `Analytics.shared.isOptedOut` played and how it's different to `disableAnalyticsSharing`. Do you know?

Reply:
`isOptedOut` is not used anymore and from what I can found here's how the getter and the setter is looking like:

```swift
import Foundation
import WireDataModel

extension Analytics {

    /// Opt the user out of sending analytics data
    var isOptedOut: Bool {
        get {
            guard let provider else { return true }
            return provider.isOptedOut
        }

        set {
            if newValue && (provider?.isOptedOut ?? false) {
                return
            }

            if newValue {
                provider?.flush {
                    self.provider?.isOptedOut = newValue
                    self.provider = nil
                }
            } else {
                provider = AnalyticsProviderFactory.shared.analyticsProvider()

                if let user = SelfUser.provider?.providedSelfUser {
                    selfUser = user
                } else {
                    assertionFailure("expected available 'user'!")
                    selfUser = nil
                }
            }
        }
    }
}
```